bootstrap.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+echo -e "\ninstalling required software packages...\n"
+zypper -q -n install update-alternatives ruby-devel make gcc gcc-c++ libxml2-devel libxslt-devel nodejs screen mariadb libmysqld-devel sqlite3-devel imagemagick
+
+echo -e "\ndisabling versioned gem binary names...\n"
+echo 'install: --no-format-executable' >> /etc/gemrc
+
+echo -e "\ninstalling bundler...\n"
+gem install bundler
+
+echo -e "\ninstalling your bundle...\n"
+su - vagrant -c "cd /vagrant/; bundle install --quiet"
+
+# Configure the app if it isn't
+if [ ! -f /vagrant/config/options.yml ] && [ -f /vagrant/config/options.yml.example ]; then
+  echo "Configuring your app in config/options.yml..." 
+  cp config/config.yml.example config/config.yml
+  cp config/secrets.yml.example config/secrets.yml
```

Comment:
If you're not checking if the file exists before this will overwrite the file if it exists already...